<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderPropertyGridWidget</class>
 <widget class="QWidget" name="SliderPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>89</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QDoubleSpinBox" name="maxValueSpin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="maxValueLabel">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>65</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="minValueLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Value:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minValueSpin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valueSpin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="valuePosition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>valuePosition</tabstop>
  <tabstop>valueSpin</tabstop>
  <tabstop>minValueSpin</tabstop>
  <tabstop>maxValueSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
